content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md

```diff
@@ -3,126 +3,122 @@ title: Impact of the CircleCI Security Incident on the Datadog Agent
+On January 4th, 2023, Datadog was notified by CircleCI that they were investigating a [security incident][1] that may have led to leaking of stored secrets. Datadog identified a single secret stored in CircleCI that could theoretically be misused by a potential attacker, an old RPM GNU Privacy Guard (GPG) private signing key and its passphrase. This page provides information about the implications of the potential leak, actions you should take on your hosts, and the measures Datadog is taking to mitigate any risks to our customers.
 
 <div class="alert alert-info">
-<strong>Note</strong>: As of January 16th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking the following actions out of an abundance of caution.
+<strong>Note</strong>: As of January 16th, 2023, Datadog has no indication that the key was actually leaked or misused, but we are still taking and advising the following actions out of an abundance of caution.
 </div>
 
 ## The affected key
 
 The impacted RPM GPG signing key has the fingerprint `60A389A44A0C32BAE3C03F0B069B56F54172A230`, and is accessible in [our signing keys location][2]. This key was historically used to sign:
 
-* Agent 5 releases and Agent 6 releases up to (and including) v6.13.0 (`datadog-agent` package)
-* Standalone DogStatsD 6 releases and Standalone DogStatsD 7 releases up to (and including) v7.23.1 (`datadog-dogstatsd` package)
+* Agent 5 releases and Agent 6 releases up to and including v6.13.0 (`datadog-agent` package)
+* Standalone DogStatsD 6 releases and Standalone DogStatsD 7 releases up to and including v7.23.1 (`datadog-dogstatsd` package)
 
 <div class="alert alert-info">
-<strong>Note</strong>: Official Datadog repositories were <strong>not</strong> compromised. The signing key, if actually leaked, could be used to construct an RPM package that looks like it is from Datadog but it would not be enough to place such a package in our official package repositories. A hypothetical attacker with the affected key would need to be able upload the constructed RPM package to a repository used by the system.
+<strong>Note</strong>: Official Datadog repositories were <strong>not</strong> compromised. The signing key, if actually leaked, could be used to construct an RPM package that looks like it is from Datadog, but it would not be enough to place such a package in our official package repositories. To be effective, a hypothetical attacker with the affected key would need to be able upload the constructed RPM package to a repository used by hosts.
```

Comment:
Might be a little clearer since we're talking about the impact on users? (feel free to disregard)

```suggestion
<strong>Note</strong>: Official Datadog repositories were <strong>not</strong> compromised. The signing key, if actually leaked, could be used to construct an RPM package that looks like it is from Datadog, but it would not be enough to place such a package in our official package repositories. To be effective, a hypothetical attacker with the affected key would need to be able upload the constructed RPM package to a repository used by your hosts.
```